test/hw1/test_task1.py

```diff
@@ -0,0 +1,165 @@
+from All_home_works.hw1.task1 import check_fibonacci
+
+
+def test_check_valid_data():
+    data = (
+        0,
+        1,
+        1,
+        2,
+        3,
+        5,
+        8,
+        13,
+        21,
+        34,
+        55,
+        89,
+        144,
+        233,
+        377,
+        610,
+        987,
+        1597,
+        2584,
+        4181,
+    )
+    assert check_fibonacci(data)
+
+
+def test_check_valid_data__not_from_begin():
+    data = (
```

Comment:
I'm pretty sure that you just copy-pasted these values from the output of your code.

Test that checks that one run of code ie equals another run of the same code only test that your output is stable. It has nothing with checking if the output is correct.